plugins/opusCodec/src/OpusEncoder.cpp

```diff
@@ -0,0 +1,269 @@
+
+#include <PerfStat.h>
+#include <QtCore/QLoggingCategory>
+
+#include "OpusEncoder.h"
+#include "opus/opus.h"
+
+static QLoggingCategory encoder("AthenaOpusEncoder");
+
+static QString error_to_string(int error) {
```

Comment:
please write function name in mixed case  CS 1.1.5  https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#115-methods-or-functions-must-be-verbs-and-written-in-mixed-case-starting-with-lower-case